make/autoconf/hotspot.m4

```diff
@@ -74,6 +78,10 @@ AC_DEFUN_ONCE([HOTSPOT_SETUP_JVM_VARIANTS],
   JVM_VARIANTS=`$ECHO $JVM_VARIANTS_OPT | $SED -e 's/,/ /g' -e 's/minimal1/minimal/'`
   AC_MSG_RESULT([$JVM_VARIANTS])
 
+  if test "x$OPENJDK_TARGET_OS" = xios; then
+    VALID_JVM_VARIANTS="zero"
```

Comment:
This should probably be guarded with a check on Device versus simulator. When running on an iOS simulator, it is technically possible to dynamically generate AND execute code, hence this is not restricted to zero.

Reply:
Ah, I was not aware that one could run server on the iOS simulator. I can remove this part since my assumption seems to be incorrect.